hydra.gemspec

```diff
@@ -23,18 +23,16 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license = 'APACHE2'
 
-  gem.add_dependency 'fcrepo_wrapper', '~> 0.5.1'
-  gem.add_dependency 'solr_wrapper', '~> 0.15.0'
-  gem.add_dependency 'active-fedora', '~> 10.0.0'
-  gem.add_dependency 'hydra-head', '~> 10.0.0'
-  gem.add_dependency 'rails', '~> 4.2'
-  gem.add_dependency 'om', '~> 3.1.1'
+  gem.add_dependency 'fcrepo_wrapper', '~> 0.7'
```

Comment:
The idea with the hydra gem was to tightly pin the dependencies, such that they wouldn't change when you do `bundle update`.  That why the previous version was pinned pessimistically. This allowed you to get bug fixes, but not new features.  I think still makes sense to continue that practice. What do you think?